star-rx72n-firmware/lib/rx_hal/src/rx_mpc.c

```
@@ -302,43 +315,47 @@ rx_err_t rx_mpc_set_mtu_encoder(gpio_pin_t pin)
 {
   /* SCI pins use PSEL = 0x0A
    * TX and RX use the same PSEL code
    */
   (void)is_tx; /* Not used - same PSEL for TX/RX */
-  return rx_mpc_set_peripheral(pin, k_psel_sci_tx);
+  const rx_mpc_peripheral_config_t config = {.pin = pin, .psel = k_psel_sci_tx};
+  return rx_mpc_set_peripheral(&config);
 }
 
-rx_err_t rx_mpc_set_riic(gpio_pin_t pin, bool is_scl)
+rx_err_t rx_mpc_set_riic(rx_port_pin_t pin, bool is_scl)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_mpc_set_riic' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)